addons/survey/views/survey_question_views.xml

```diff
@@ -171,23 +171,30 @@
+                                               attrs="{'invisible': [('question_type', '!=', 'char_box')], 'required': [('validation_required', '=', True), ('question_type', '=', 'char_box')]}"/>
+                                        <field name="validation_min_float_value" class="o_survey_question_validation_short" nolabel="1" placeholder='Minimum'
+                                               attrs="{'invisible': [('question_type', '!=', 'numerical_box')], 'required': [('validation_required', '=', True), ('question_type', '=', 'numerical_box')]}"/>
+                                        <field name="validation_min_date" nolabel="1" placeholder="Minimum"
+                                               attrs="{'invisible': [('question_type', '!=', 'date')], 'required': [('validation_required', '=', True), ('question_type', '=', 'date'), ('validation_max_date', '=', False)]}"/>
+                                        <field name="validation_min_datetime" nolabel="1" placeholder="Minimum"
+                                               attrs="{'invisible': [('question_type', '!=', 'datetime')], 'required': [('validation_required', '=', True), ('question_type', '=', 'datetime'), ('validation_max_datetime', '=', False)]}"/>
 
+                                        <i class="fa fa-long-arrow-right mx-2" aria-label="Arrow icon" title="Arrow"/>
+                                        <!-- Maxima -->
+                                        <field name="validation_length_max" class="o_survey_question_validation_short" nolabel="1" placeholder='Maximum'
+                                               attrs="{'invisible': [('question_type', '!=', 'char_box')], 'required': [('validation_required', '=', True), ('question_type', '=', 'char_box')]}"/>
+                                        <field name="validation_max_float_value" class="o_survey_question_validation_short" nolabel="1" placeholder='Maximum'
+                                               attrs="{'invisible': [('question_type', '!=', 'numerical_box')], 'required': [('validation_required', '=', True), ('question_type', '=', 'numerical_box')]}"/>
+                                        <field name="validation_max_date" nolabel="1" placeholder="Maximum"
+                                               attrs="{'invisible': [('question_type', '!=', 'date')], 'required': [('validation_required', '=', True), ('question_type', '=', 'date'), ('validation_min_date', '=', False)]}"/>
+                                        <field name="validation_max_datetime" nolabel="1" placeholder="Maximum"
+                                               attrs="{'invisible': [('question_type', '!=', 'datetime')], 'required': [('validation_required', '=', True), ('question_type', '=', 'datetime'), ('validation_min_datetime', '=', False)]}"/>
+                                    </div>
```

Comment:
could we make this multi line ?

Reply:
Definitely! I hadn't seen it in the codebase and thought it would break, but no :-)